.env.dev

```diff
@@ -6,4 +6,7 @@ DB_NAME=activity_dev
 DB_USER=activity
 DB_PASSWORD=activity
 DB_HOST=db
-DB_PORT=5432
\ No newline at end of file
+DB_PORT=5432
+
+SOCIAL_AUTH_GOOGLE_OAUTH2_KEY=840126614114-r1pucia7mshjatscj09dm40qa4mpd014.apps.googleusercontent.com
```

Comment:
Remove key and secret

Reply:
:man_facepalming:

Reply:
@Darius-Ndubi can you remove these credentials if you're making some adjustments to prepare this for our deployment environment?